usr/sbin/registercloudguest

```diff
@@ -57,7 +57,7 @@ registration_returncode = 0
 # ----------------------------------------------------------------------------
 def register_modules(
     extensions, products, registration_target, instance_data_filepath,
-    registered=[], failed=[]
+    regcode, registered=[], failed=[]
```

Comment:
The default value for `regcode` in `register_product` is set to `regcode=''`. As this code just passes along the regcode to another method I recommend to use the same default value

```python
regcode=''
```

for consistency reasons

Reply:
Done